.travis.yml

```diff
@@ -723,6 +721,18 @@ matrix:
             - export JAVA_HOME=$TEST_JDK
             - hide-logs.sh ant $OPTS commit-validation
 
+#   exile: for tests which don't behave. Job should be kept short to be easily restartable
+        - name: Test unreliable tests on Java 8
+          jdk: openjdk8
+          env:
+            - OPTS="-Dmetabuild.jsonurl=https://raw.githubusercontent.com/apache/netbeans-jenkins-lib/master/meta/netbeansrelease.json -quiet -Dcluster.config=java -Djavac.compilerargs=-nowarn -Dbuild.compiler.deprecation=false -Dtest-unit-sys-prop.ignore.random.failures=true"
+          before_script:
+            - nbbuild/travis/ant.sh $OPTS clean
+            - nbbuild/travis/ant.sh $OPTS build
+          script:
+            - travis_retry ant $OPTS -f java/java.lsp.server test
+            - travis_retry hide-logs.sh ant $OPTS -f java/gradle.java test
```

Comment:
The "hide-logs.sh" filter in the second line will also show the failed tests of the previous "java/java.lsp.server" test, right? I understand this is on purpose, not a typo, right?

Reply:
> Regarding retries: if tests are reliable there's no need to retry them, right? :-).

right. Thats why I wanted to wait till your PR is integrated to check how the improved test behaves. 

Btw `travis_retry` is used in 14 other places in that travis.yml.